### ISSUE TITLE
Event `LeftOpen` shows wrong `timeOpen100ms`

### ISSUE THREAD

**Issue by runely** [remootio/remootio-api-client-node#10]
The **Gate status sensor** is connected and configured to notify if gate is left open for **3600** seconds.

The notification sent from Remootio app on my phone shows correctly that my Remootio device has been open for **60 minutes 0 seconds**

But the API shows me that it has been open for **300 seconds**. The API will actually tell me that it has been open for **300 seconds** no matter what *notify if gate is left open for* is configured to.

This happens on both my **Remootio2** and **Remootio3** device. They are both running on firmware version **2.27**. And the node API is **2.0.3**

`Event from API`
```json
{
  "event": {
    "cnt": 18,
    "type": "LeftOpen",
    "state": "open",
    "t100ms": 1015708,
    "data": {
      "timeOpen100ms": 3000
    }
  }
}
```

`Notification from Remootio app`

![image](https://user-images.githubusercontent.com/25814673/133882524-5c721351-3270-4027-b465-1fe60929ca23.png)

**Comment by remootio** [comment]
@runely Thanks for reporting this issue. This is a bug on the Remootio's side. We'll fix it in the next release

**Comment by remootio** [comment]
Fix will be released in Remootio SW 2.29

**Comment by runely** [comment]
Any ETA when 2.29 will be released?

**Comment by runely** [comment]
I can confirm this is resolved in SW `2.28` 👍
